### PR TITLE
Consistently reference `theme_root_icon`

### PR DESCRIPTION
**Pull request by mbudisic**
Removed explicit `py.svg` references in `layout.html` with `theme_root_icon` paths. 

This pull request changes paths lines 17 and 103 to match line 74 in the base branch.

### CODE REVIEW

**Review by ezio-melotti — APPROVED** [review]

**Comment by hugovk** [comment]
Good idea.

Although "Logo" isn't a great alt text, can we also add something like `root_icon_alt_text` to the config?

* https://webaim.org/techniques/alttext/#logos
* https://sc.edu/about/offices_and_divisions/digital-accessibility/toolbox/best_practices/alternative_text/logo-alt-text/index.php
* https://www.a11yproject.com/posts/alt-text/

File: python_docs_theme/theme.conf [hunk 1]
```diff
@@ -32,4 +32,5 @@ root_name = Python
 root_url = https://www.python.org/
 root_icon = py.svg
 root_include_title = True
+root_icon_alt_text = Python Logo
```
Comment (hugovk) [inline python_docs_theme/theme.conf:35]:
```suggestion
root_icon_alt_text = Python logo
```

Comment (mbudisic) [reply python_docs_theme/theme.conf:35]: Thank you.

**Review by hugovk — APPROVED** [review]
Thanks!

File: python_docs_theme/theme.conf [hunk 2]
```diff
@@ -32,4 +32,5 @@ root_name = Python
 root_url = https://www.python.org/
 root_icon = py.svg
 root_include_title = True
+root_icon_alt_text = Python logo
```
Comment (ezio-melotti) [inline python_docs_theme/theme.conf:35]: Any reason why the two `root_icon*` are not together?
```suggestion
root_icon = py.svg
root_icon_alt_text = Python logo
root_include_title = True
```

File: python_docs_theme/layout.html [hunk 3]
```diff
@@ -14,7 +14,7 @@ <h3>{{ _('Navigation') }}</h3>
           </li>
           {%- endfor %}
           {%- block rootrellink %}
-          <li><img src="{{ pathto('_static/py.svg', 1) }}" alt="python logo" style="vertical-align: middle; margin-top: -1px"/></li>
+          <li><img src="{{ pathto('_static/' + theme_root_icon, 1) }}" alt="{{theme_root_icon_alt_text}}" style="vertical-align: middle; margin-top: -1px"/></li>
```
Comment (AA-Turner) [inline python_docs_theme/layout.html:17]:
```suggestion
          <li><img src="{{ pathto('_static/' ~ theme_root_icon, 1) }}" alt="{{theme_root_icon_alt_text}}" style="vertical-align: middle; margin-top: -1px"/></li>
```

File: python_docs_theme/layout.html [hunk 4]
```diff
@@ -100,7 +100,7 @@ <h3>{{ _('Navigation') }}</h3>
         </label>
         <span class="nav-items-wrapper">
             <a href="{{ theme_root_url }}" class="nav-logo">
-                <img src="{{ pathto('_static/py.svg', 1) }}" alt="Logo"/>
+                <img src="{{ pathto('_static/' + theme_root_icon, 1) }}" alt="{{theme_root_icon_alt_text}}"/>
```
Comment (AA-Turner) [inline python_docs_theme/layout.html:103]:
```suggestion
                <img src="{{ pathto('_static/' ~ theme_root_icon, 1) }}" alt="{{theme_root_icon_alt_text}}"/>
```

**Review by AA-Turner — CHANGES_REQUESTED** [review]
Per the [documentation](https://jinja.palletsprojects.com/en/3.1.x/templates/#other-operators):

> This, however, is not the preferred way to concatenate strings! For string concatenation, have a look-see at the `~` operator.

https://jinja.palletsprojects.com/en/3.1.x/templates/#other-operators

> Converts all operands into strings and concatenates them.

A

**Pull request merged**